src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -414,4 +430,88 @@ private void shutdown(long endTime) {
             return readers;
         }
     }
+
+    static private class KafkaTest extends Test {
+        final private Properties producerConfig;
+        final private Properties consumerConfig;
+
+        KafkaTest(long startTime, CommandLine commandline) throws
+                IllegalArgumentException, URISyntaxException, InterruptedException, Exception {
+            super(startTime, commandline);
+            producerConfig = createProducerConfig();
+            consumerConfig = createConsumerConfig();
+        }
+
+        private Properties createProducerConfig() {
+            if (producerCount < 1) {
+                return null;
+            }
+            final Properties props = new Properties();
+            props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, controllerUri);
+            props.put(ProducerConfig.ACKS_CONFIG, "all");
+            props.put(ProducerConfig.RETRIES_CONFIG, 0);
+            props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
```

Comment:
Are the serializers being used for Pravega and Kafka equivalent? It would be good to use the same type of serializer, given that it would yields the same serialization overhead for both systems, which is fair.

Reply:
yes.. its same as UTF8String sterilizer used in pravega and for Kafka too: https://kafka.apache.org/0100/javadoc/org/apache/kafka/common/serialization/StringSerializer.html